Desserts/Data/Helpers/Helpers.swift

```diff
@@ -47,20 +47,22 @@ extension Bundle {
                 }
             }.resume()
     }
-    func encode<T: Encodable>(data: T, to file: String) {
-            let encoder = JSONEncoder()
+    func encode<T: Encodable>(data: T, to file: String, key: String) {
+        let existingObjects = [key: data]
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = .prettyPrinted
+        do {
             
-            do {
-                let jsonData = try encoder.encode(data)
-                guard let url = self.url(forResource: file, withExtension: "json") else {
-                    fatalError("Could not find \(file) in bundle.")
-                }
-                
-                try jsonData.write(to: url)
-                print("Data saved successfully to \(file).json")
-            } catch {
-                print("Encoding error: \(error)")
-                fatalError("Could not encode data to \(file).json")
+            let jsonData = try encoder.encode(existingObjects)
+            var jsonString = String(data: jsonData, encoding: .utf8)!
```

Comment:
If you're using a `guard` in the next line, then I suggest to take advantage of that to include this `String` creation as part of the `guard` conditions. Just to avoid to deal with force-unwrapping statements

Reply:
Done!